App/TopTrumps/TopTrumps.Data/Helpers/SqlHelper.cs

```diff
@@ -11,7 +11,7 @@ public string GetCardQuery(int id)
             return $@"SELECT * FROM [Cards] WHERE [Id] = @{nameof(id)}";
         }
 
-        public string GetAllCardsQuery()
+        public string GetAllUsersQuery()
```

Comment:
Why are we renaming the `GetAllCardsQuery` to `GetAllUsersQuery` ?

The SQL that this method returns is for cards not users. Did you mean to create a new method similar to this one but for users? You have actually done this below with the `GetAllUserssql` method below.

Reply:
Mistake. Meant to use similar method and must have over written this name.